### DISCUSSION TITLE
Splits - Split transactions

### DISCUSSION THREAD

**Discussion by garters** [moneymanagerex/moneymanagerex#4883] [Ideas]
Im wondering why it doesn't automatically calculate when you split a transaction. Eg start with $240; click split button; change first category to power and $200; next line has $40 waiting ready for you to choose the next category. And so on down the split list. That way would perhaps avoid calculation errors.

**Comment by tactilis** [comment]
+1
Nice idea @garters

**Reply by markd121** [reply]
This used to be a feature until recently - within the last 4-5 updates.  Was very helpful when entering split transactions.  Not sure why it disappeared
